app/socksMysql.rb

```diff
@@ -52,12 +52,27 @@ def service(ready)
 
   #################################################################################################
   def bidiTransfer(localSock, remoteSock)
+    first = true
     while true
       rds, wrs, ers = IO.select([localSock, remoteSock], [])
       rds.each { |r|
         data = r.recv(8192)
         data.empty? and return
-        #puts "Tranferring data from #{r == localSock ? "local" : r == remoteSock ? "remote" : r}: #{data.inspect}"
+        if first
```

Comment:
Can we make this change separately to have the change history more accurately reflect the reason for the change.

Reply:
sure, I'll untangle this

Reply:
untangled: https://github.com/eScholarship/jschol/pull/625